### ISSUE TITLE
Links to guidance pages are not always clearly marked 

### ISSUE THREAD

**Issue by YedidaZalik** [cds-snc/notification-planning#1217]
As a user, I need to be able to easily identify when content provides a link to guidance pages. 

WHY are we building?
Our guidance uses descriptive links, but it's not always clear that we are linking to Guidance pages, especially for Ally, screen-readers but also users who are not using assistive tech.

WHAT are we building?
We're going to audit references to the guidance in the UI to make sure that these are clearly indicated.
Some of this may be fixed by bounce rate content for v1.5.

VALUE created by our solution

## Documentation and Artifacts

Good docs, figma mockups, ADRs, screenshots etc.

## Acceptance Criteria
When content includes hyperlinks,  we consider all aspects of  issues in the way that we present link text.
Note that during this sprint we learned of possible new best practices with respect to link text.

- [x] Make it easier for all users to identify hyperlinks in Notify guidance docs
- [x] Make it easier for users to identify when hyperlinks will lead to a new guidance page or more specific guidance so that users can decide whether to leave the page that they are on
- [x] Consult to determine how to assess whether to present link text at the end of sentences or to embed within sentences
- [ ] Inventory of places that hyperlink to guidance so that we can update those links with the new titles

* A11y
* Bilingualism
* Privacy considerations
* Security controls in place
* Measuring success and metrics

## Related [Research Airtable](https://airtable.com/appWwAmHwDLtpIyko/tbl38n7ofWYBuezFc/viwQUX3tUxbMxYS7C?blocks=hide) records
- (Ping Adrianne to add stuff here or find stuff yourself)

## QA Steps
- [ ] Tested in a realistic production scenario

**Comment by yaelberger-commits** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Notify-Planning-614b3ad91bc2030015ed22f5/issues/cds-snc/notification-planning/1217?planning-poker) with Zenhub @YedidaZalik @amazingphilippe

**Comment by anikbrazeau** [comment]
For context, here's what our current iteration of in-line guidance looks like in GC Forms (by no means a standard):

![image.png](https://images.zenhubusercontent.com/612f6a0f0d148b7f2b0a72b2/256d6e1f-7344-4472-9bc2-9e8219031bc5)

![image.png](https://images.zenhubusercontent.com/612f6a0f0d148b7f2b0a72b2/3d506855-2b58-43f6-9341-45dba3127e8f)

![image.png](https://images.zenhubusercontent.com/612f6a0f0d148b7f2b0a72b2/4aa223ca-fee1-41c2-851f-cabe574e450d)

Originally this was experimented with in a modal, but now we're trying to do so in an accordion instead, making the information more secondary to the main content / service flow.

**Comment by YedidaZalik** [comment]
Met about this yesterday and started work in this document: https://docs.google.com/document/d/1EuhmtD71Xe1mwTIAIUi8XHaQZGPesgeySP5G_aFPaNI/edit#heading=h.j0seccafwdvo

**Comment by anikbrazeau** [comment]
The [Details component](https://design-system.alpha.canada.ca/en/components/details/) on GC Design system // [Expand/Collapse](https://design.canada.ca/common-design-patterns/collapsible-content.html) on canada.ca might serve as some inspiration --- used to present secondary information.

**Comment by andrewleith** [comment]
- Guidelines around links are a bit ambiguous; reaching out to other experts at IDRC to determine the best approach
- Mario to provide a contact from the ACT group at SSC

**Comment by YedidaZalik** [comment]
We do not yet have a definitive decision on links. We've tagged design system and Anik  i[n the platform design channel for guidance](https://gcdigital.slack.com/archives/C01UG1A4QE9/p1687801232277129).
For the moment, we are mostly using links at the end of sentences, but will use embedded links in 2 situations where it seems more natural:

![Screenshot 2023-06-27 at 10.21.51 AM.png](https://images.zenhubusercontent.com/614b9f22b18cf806c628a310/6e0410ee-7069-4702-b632-aec6ad70ad7c)
![Screenshot 2023-06-27 at 10.21.38 AM.png](https://images.zenhubusercontent.com/614b9f22b18cf806c628a310/263e10a3-c9c3-48ab-ab24-a9efbc8d1678)

**Comment by YedidaZalik** [comment]
Very helpful reply from Lisa at IDRC


Thanks for your question, and for sharing Perston So’s excellent talk!

 

When trying to design for a range of needs and goals, there is often no simple/single answer. There are pros and cons to the different approaches that work for different needs. Often, they are flip sides of the same coin. Let’s take Preston’s example of a paragraph with relevant links in it. One benefit of peppering links throughout text is that navigating to other content and away from the main text is quick and easy, and can be done in the moment, rather than having to finish a paragraph and then find the relevant link. The downside is that this also makes content more difficult to focus on and may be distracting. Having a lot of underlined text within a paragraph also negatively impacts readability from a visual standpoint (I believe this is one reason why Wikipedia doesn’t underline links within its content even though that makes the site fail SC 1.4.1 Use of Color). In this example, I would consider how the reader is meant to engage with the information. If it requires deeper reading, placing the links after the information may be better to reduce distractions and improve readability. If the goal of the paragraph is to provide context for the links and serve as effective redirection to other pages, then links within the text could be a better approach, especially for screen reader and braille display users. I hope that helps!

 

One more thought that I wanted to share as feedback about Preston’s talk: I noticed that a couple of the examples that Preston gives of the reworded link text didn’t provide enough clarity about the link destination, in my opinion. For example, in “Medicaid has several programs that can help you and your family”. It’s not clear what programs the link is referring to from the link alone (particularly because the site likely covers many types of programs, not just Medicaid). One of the things he doesn’t get into in the talk is that screen reader and braille display users often navigate through only the links on the page, meaning they don’t always hear/feel the entire sentence or paragraph that contains a link. As much as possible the link text should be a self-contained description of the link destination. I think “There are several Medicaid programs that can help you and your family” would have been a better choice.

 

Please let me know if you have other questions, or if you’d like to chat about this further.

 

Kind regards,

Lisa

**Comment by YedidaZalik** [comment]
Lisa's comments fit with what we've done for the guidance this time round.
Document is ready for translation.

**Comment by YedidaZalik** [comment]
The document is ready for French content by @msbcsc

**Comment by YedidaZalik** [comment]
Created [inventory of pages that hyperlink to guidance](https://docs.google.com/document/d/1N6B4fSwoM40m-pDLI70BvLfL7O0n04ndyAXYaIwNbNw/edit) as we'll have to update links for those that do

**Comment by adriannelee** [comment]
Meeting with Marie-Sophie today to discuss translation

**Comment by yaelberger-commits** [comment]
Phil working on URLs PR
